docs/releasenotes/index.rst

```diff
@@ -16,6 +16,7 @@ Next Release
 * The new :ref:`CloudFormation installer <cloudformation>` has been made available, to provide a far simpler installation experience for users on AWS.
 * The :ref:`Flocker plugin for Docker <plugin>` should support the direct volume listing and inspection functionality added to Docker 1.10.
 * Fixed a regression that caused block device agents to poll backend APIs like EBS too frequently in some circumstances.
+* Datasets can now be moved off of shutdown EC2 instances. Previously they could only be moved off of terminated instances.
```

Comment:
Right, but is it a matter of hoping that no one tries it? If so are we
happy with that?

On Fri, 29 Jan 2016 at 14:31, Itamar Turner-Trauring <
notifications@github.com> wrote:

> In docs/releasenotes/index.rst
> <https://github.com/ClusterHQ/flocker/pull/2546#discussion_r51266013>:
>
> > @@ -16,6 +16,7 @@ Next Release
> >  * The new :ref:`CloudFormation installer <cloudformation>` has been made available, to provide a far simpler installation experience for users on AWS.
> >  * The :ref:`Flocker plugin for Docker <plugin>` should support the direct volume listing and inspection functionality added to Docker 1.10.
> >  * Fixed a regression that caused block device agents to poll backend APIs like EBS too frequently in some circumstances.
> > +* Datasets can now be moved off of shutdown EC2 instances. Previously they could only be moved off of terminated instances.
>
> I've tried it on Rackspace manually. It doesn't work due to Rackspace
> bugs. It *does* work on OpenStack elsewhere (at least on Swisscom's
> cluster) but I don't want to claim that publicly until we can enable the
> acceptance tests.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ClusterHQ/flocker/pull/2546/files#r51266013>.
>